src/audio/pipeline.c

```diff
@@ -418,7 +415,6 @@ int pipeline_prepare(struct pipeline *p, struct comp_dev *dev)
 	p->status = COMP_STATE_PREPARE;
 
 out:
-	irq_local_enable(flags);
```

Comment:
We don't need `out` label anymore. You can return immediately on error.

Reply:
Ok, I agree we can remove it. Done.